components/brave_rewards/browser/test/rewards_publisher_browsertest.cc

```diff
@@ -147,6 +147,18 @@ IN_PROC_BROWSER_TEST_F(RewardsPublisherBrowserTest, VisitUnverifiedPublisher) {
       false);
 }
 
+// Registered publishers without a wallet address are displayed as verified
+IN_PROC_BROWSER_TEST_F(RewardsPublisherBrowserTest, VisitRegisteredPublisher) {
+  rewards_browsertest_helper::EnableRewards(browser());
```

Comment:
please use enable via code

Reply:
The following call `VisitPublisher` expects the current tab to have the rewards page loaded, so using `EnableViaCode` will require us to load that page and wait first.

For consistency with the other `Visit*Publisher` tests, would it be OK to leave this as-is?

Reply:
sound good